src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars stars--0">
```

Comment:
Don't add empty lines between parents and child
```suggestion
    <div class="container">
      <div class="stars stars--0">
```